src/sagas/index.js

```diff
@@ -48,6 +48,7 @@ function* rootSaga() {
 
         break
       } catch (e) {
+        log.error(e);
```

Comment:
[fix] This method signature needs a message as first param. You can combine with the log bellow.
```js
log.error('...log bellow', e);
```

Reply:
Done, thanks